data/id/legislature/Steve-Miller-7efc6ba2-1301-417b-9745-8505520c49cc.yml

```diff
@@ -0,0 +1,23 @@
+id: ocd-person/7efc6ba2-1301-417b-9745-8505520c49cc
+name: Steve Miller
+given_name: Steve
+family_name: Miller
+email: SMiller@house.idaho.gov
+image: https://legislature.idaho.gov/wp-content/uploads/sessioninfo/2023/directory/Miller2134.jpg
+party:
+- name: Republican
+roles:
```

Comment:
@bfossen-ce this person served in the house district 26a between 12/01/2012 and 12/01/2018.